buildSrc/src/main/java/Dependencies.kt

```diff
@@ -55,6 +55,9 @@ object Dependencies {
     object gradle {
         val androidPlugin = "com.android.tools.build:gradle:3.5.3"
         val kotlinPlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlin}"
+        object avito {
+            val kotlinDslSupport = "com.avito.android:kotlin-dsl-support"
```

Comment:
Minor change. Need to use in buildscript. I don't know how use infraVersion in buildSrc